File: help/pt_pt/exercicios_da_ficha_2/Ficha02_011/Ficha02_011.java
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha02_011 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        int n1 = 0 ;
        int n2 = 0 ;
        System.out.print("1º número : " );
        n1 = Integer.valueOf(keyb.nextLine());
        System.out.print("2º número : " );
        n2 = Integer.valueOf(keyb.nextLine());
        int soma = n1 + n2 ;
        System.out.print(n1 + " + " + n2 + " = " + soma );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 set 07 16:11

File: help/eng_gb/exercicios_da_ficha_2/Ficha02_011/Ficha02_011.php
<?php
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    $n1 = 0 ;
    $n2 = 0 ;
    echo  "1st number : " ;
    $n1 = intval(isset($_POST["n1"])? $_POST["n1"] : isset($_GET["n1"]) ? $_GET["n1"] : 0);
    
    echo $n1 . "<br/>";// print input 
    echo  "2nd number : " ;
    $n2 = intval(isset($_POST["n2"])? $_POST["n2"] : isset($_GET["n2"]) ? $_GET["n2"] : 0);
    
    echo $n2 . "<br/>";// print input 
    $sum = $n1 + $n2 ;
    echo $n1  .  " + " . $n2  .  " = " . $sum  ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	quarta, 2022 set 21 22:52

?>